mixnet/src/crypto.rs

```diff
@@ -40,6 +40,26 @@ impl Default for PrivateKey {
     }
 }
 
+impl Serialize for PrivateKey {
+    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: Serializer,
+    {
+        SerializablePrivateKey::from(self).serialize(serializer)
+    }
+}
```

Comment:
This can be derived by `serde(transparent)`.

```rust
#[derive(Serialize, Deserialize)]
#[serde(transparent)]
pub struct PrivateKey(sphinx_packet::crypto::PrivateKey);
```